docs/book/v3/helpers/doctype.md

```diff
@@ -18,38 +17,31 @@ The `Doctype` helper allows you to specify one of the following types:
 - `HTML4_LOOSE`
 - `HTML4_FRAMESET`
 - `HTML5`
-- `CUSTOM_XHTML`
-- `CUSTOM`
-
-You can also specify a custom doctype as long as it is well-formed.
-
-The `Doctype` helper is a concrete implementation of the
-[Placeholder helper](placeholder.md).
 
 ## Basic Usage
 
-You may specify the doctype at any time. However, helpers that depend on the
-doctype for their output will recognize it only after you have set it, so the
-easiest approach is to specify it in your bootstrap:
+The `Doctype` helper requires a constant indicating the desired doctype to its constructor.
+It can then be cast to a string in order to emit the doctype declaration:
 
 ```php
 use Laminas\View\Helper\Doctype;
 
-$doctypeHelper = new Doctype();
-$doctypeHelper->doctype('XHTML1_STRICT');
+$helper = new Doctype(Doctype::HTML5);
```

Comment:
HTML5 is the default doctype therefore it should be omitted.
A second example with a different doctype would be good.